sketch/tensor.hpp

```diff
@@ -102,12 +111,21 @@ class Tensor {
         return result;
     }
 
+    double discretize(double val) {
+//        auto bin = std::upper_bound(bins.begin(), bins.end(), val) - bins.begin();
+//        auto bin = atan(val);
```

Comment:
I wouldn't publish code like this on github. I suggest completely removing the binning, as it does not appear in the paper. 
If you'd like to add it, then add it nicely, perhaps as a new BinnedTensorSketch class.